docker/dockerfiles/Dockerfile.server

```diff
@@ -18,6 +18,7 @@ RUN apt-get update; apt-get install -y \
     zip;
 
 # Install dependencies
+COPY docker/compose_files/files/wait-for-it.sh /data/bin/wait-for-it.sh
```

Comment:
why `/data`?  seems like a semantically odd choice

Reply:
This is where it was placed in the old deploy script I started from months ago but I agree, I'll probably put everything in `/opt`